test/common/router/config_impl_speed_test.cc

```diff
@@ -27,11 +27,11 @@ using testing::ReturnRef;
  * Generates a request with the path:
  * - /shelves/shelf_x/route_x
  */
-static Http::TestRequestHeaderMapImpl genRequestHeaders(int route_num) {
+static Http::TestRequestHeaderMapImpl genRequestHeaders(int shelf_id, int route_num) {
```

Comment:
Seems that the shelf_id always equal to route_num?

Reply:
Agreed. I'll remove the `shelf_id` param and will keep it as before.